src/core/reactor.cc

```diff
@@ -481,7 +481,7 @@ future<size_t> pollable_fd_state::recvmsg(struct msghdr *msg) {
         speculate_epoll(EPOLLIN);
         return make_ready_future<size_t>(*r);
     });
-};
+}
```

Comment:
i'd suggest s/style/reactor/ in the title of the commit message.

Reply:
good idea, the word `style` seems too general.